ports/linux/rs485.c

```diff
@@ -607,6 +607,13 @@ void RS485_Initialize(void)
             exit(EXIT_FAILURE);
         }
         /* if all goes well, set new divisor */
+        ioctl(RS485_Handle, TIOCSSERIAL, &newserial);
```

Comment:
the ioctl is common for both if and else we could move it out conditional block

Reply:
I've taken it outside of the if else statement